opta/utils/globals.py

```diff
@@ -0,0 +1,38 @@
+from typing import Set
+
+from colored import attr, fg
+
+from opta.exceptions import UserErrors
+
+
+class Interpolation:
+    bad_field_names: Set[str] = set()
+
+    @classmethod
+    def validate(cls) -> None:
+        if len(cls.bad_field_names) > 0:
```

Comment:
```suggestion
        if not cls.bad_field_names:
            return
```

Then unindent the remaining method lines. Sets are falsey when empty (as are most python collection types), and by returning early, it reduces the amount of indentation for later lines in the method, known as a "guard clause"